src/analysis.rs

```diff
@@ -27,7 +27,7 @@ pub(crate) const IO_EXTRA_FRAMES: usize = 2;
 /// Nodes with left children require their finalized left child,
 /// while nodes with right children require their finalized right child.
 /// Witness nodes require their node type.
-pub(crate) fn compute_bounds<App: Application>(
+pub(crate) fn compute_bounds<App: Jet>(
```

Comment:
In d689e20c5cfb81f6a06a15eaef3ea32aa62efe3a:

We continue to have `App: Jet` in many places. I think it may be clearer to have `J: Jet` instead. First because it gets rid of the confusing name "App" and secondly because there's a convention that (usually) generic types have 1- or 2-character names.

Reply:
Sure, I will push another commit. It's funny how we return to `Jet` which existed in this repository before I killed it.